src/main/webapp/app/shared/markdown-editor/monaco/markdown-editor-monaco.component.scss

```diff
@@ -1,8 +1,32 @@
 .markdown-editor-wrapper {
+    display: flex;
+    flex-flow: column nowrap;
+    flex: 1 1 auto;
+    height: inherit;
+    color: var(--md-wrapper-color);
     background-color: var(--md-editor-background);
+
+    // This is a fix for using the fullscreen mode on Safari.
```

Comment:
Fix for what? Is there a bug in the markdown editor?

Reply:
I copied this style from the original markdown editor's style file (which the new md editor is using). I didn't want to break the Fullscreen mode of Safari, so I didn't remove the style